src/EffectiveCSharp.Analyzers/AnalyzerReleases.Unshipped.md

```diff
@@ -1,2 +1,11 @@
 ﻿; Unshipped analyzer release
 ; https://github.com/dotnet/roslyn-analyzers/blob/main/src/Microsoft.CodeAnalysis.Analyzers/ReleaseTrackingAnalyzers.Help.md
+
+### New Rules
+
+Rule ID | Category | Severity | Notes
+--------|----------|----------|-------
+ECS1200 | Maintainability | Info | PreferDeclarationInitializersToAssignmentStatement, [Documentation](https://github.com/rjmurillo/EffectiveCSharp.Analyzers/blob/97c51a41b53d059cfcd06f0c8ce5ab178b070e6b/docs/rules/ECS1200.md)
```

Comment:
Are having these rules confusing to end users? They can't be distilled down into fewer (or one)?